docs/documentation/release_notes/topics/26_2_0.adoc

```diff
@@ -8,6 +8,19 @@ For more details, see the link:{securing_apps_token_exchange_link}#_standard-tok
 
 For information on how to upgrade from the legacy token exchange used in previous {project_name} versions, see the link:{upgradingguide_link}[{upgradingguide_name}].
 
+= Supported Fine-Grained Admin Permissions
+
+This release introduces a new supported version of Fine-Grained Admin Permissions: V2, providing enhanced flexibility and control over administrative access within realms. With this feature, administrators can define permissions for administering users, groups, clients, and roles without relying on broad administrative roles. There are some key points:
+
+* *Centralized Admin Console Management* - New `Permissions` section was introduced to allow management from a single place without having to navigate to different places in the administration console. 
+* *Resource-Specific and Global Permissions* – Permissions can be defined for individual resources (e.g., specific users or groups),  or entire resource types (e.g., all users or all groups).
+* *Explicit Operation Scoping* – Permissions are now independent, removing hidden dependencies between operations. Admins must assign each scope explicitly, making it easier to see what is granted without needing prior knowledge of implicit relationships.
```

Comment:
I would add:

* Improved manageability experience so that administrators can more easily search and evaluate permissions when building an permission model for realm resources

Perhaps we should also highlight that v2 brings the same level of access control over realm resources so people should not expect anything different than what is there in v1. However, with more flexibility to support additional realm resource types and scopes.